README.rst

```diff
@@ -40,6 +40,32 @@ Remember, if co-installing multiple CWL implementations then you need to
 maintain which implementation ``cwl-runner`` points to via a symbolic file
 system link or `another facility <https://wiki.debian.org/DebianAlternatives>`_.
 
+Running tests locally
+---------------------
+
+-  Running unnittests ``(/tests)``:
+
+.. code:: bash
+
+    pip install tox
+    tox # to run all tests
+
+-  Running conformance tests:
+
+The conformance tests are intended to test feature coverage of a CWL
+implementation. It uses the module “cwltool.cwltest” from the cwltool
+reference implementation.
+
+.. code:: bash
+
+      $ ./run_test.sh
```

Comment:
We need to tell people to check out the CWL specs repository to get a copy of the conformance tests and `run_test.sh`